### ISSUE TITLE
SyntaxError: invalid syntax on line 61

### ISSUE THREAD

**Issue by fasenderos** [meltygroup/ansible-parallel#1]
Hi, I'm trying to use your package but I get the following error:

```
File "/usr/lib/python3.7/site-packages/ansible_parallel.py", line 61
    while line := (await process.stdout.readline()).decode():
                ^
SyntaxError: invalid syntax
```

Pyhton 3.7.7

Thanks

**Comment by JulienPalard** [comment]
Hi @fasenderos, this is a Python 3.8 syntax, is it easy for you to get a Python 3.8 or would you like me to build a Python 3.7 compatible version?

**Comment by fasenderos** [comment]
@JulienPalard I'll try with Python 3.8. Thanks

**Comment by JulienPalard** [comment]
@fasenderos ok, tell if it it's OK for you. If yes I'll release with a `python_requires = >= 3.8`, else I'll rewrite this walrus to keep the project compatible with 3.6.

**Comment by JulienPalard** [comment]
I released a version compatible with 3.7 :) I bet you're not alone with this version.
